### PR TITLE
Whitenoise GZIP support. Fix #32.

### LINKED ISSUE #32: Add gzip and caching support to WhiteNoise
http://whitenoise.readthedocs.org/en/latest/django.html#add-gzip-and-caching-support

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@pmclanahan r?

File: {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py [hunk 1]
```diff
@@ -92,6 +92,7 @@
 
 STATIC_ROOT = config('STATIC_ROOT', default=os.path.join(BASE_DIR, 'static'))
 STATIC_URL = config('STATIC_URL', '/static/')
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
```
Comment (pmclanahan) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py]: This is good as long as you're not also using django-pipeline. You have to combine the classes for that [like we did in bedrock](https://github.com/mozilla/bedrock/blob/532669e9ee15c20a65cd349a15cc998b46253844/bedrock/base/storage.py).

**Comment by pmclanahan** [comment]
r+ as long as pipeline isn't involved.

File: {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py (hunk 1 above)
Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py]: Good to know. I filed #70 to discuss whether we want django-pipeline or not.

**Comment by glogiotatidis** [comment]
Thanks @pmclanahan  :)

**Pull request merged**